public/app/features/explore/spec/query.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { screen } from '@testing-library/react';
+
+import { serializeStateToUrlParam } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+
+import { makeLogsQueryResponse, makeMetricsQueryResponse } from './helper/query';
+import { setupExplore, tearDown, waitForExplore } from './helper/setup';
+
+describe('ExplorePage handles running/not running query', () => {
```

Comment:
Minor nit: In other tests we use `Explore: ` prefix but we don't have clear guidelines around it so you can also leave it as it is.

Reply:
I like "Explore: " since I was struggling with the wording. I removed the test from ExplorePage.test.tsx so referring to the ExplorePage component didn't seem right but I didn't have a better idea.